_posts/2019-02-28-newpipe-0.16.0-released.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: NewPipe 0.16.0 released, adding comment support and mediaCCC service
+short: "NewPipe 0.16.0 released"
+date: 2019-02-28 19:00:00 +0200
+categories: [release, pinned]
+author: TobiGr
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+We are happy to release NewPipe 0.16.0 bringing support for reading top-level YouTube comments. Moreover, with [media.ccc.de](https://media.ccc.de) NewPipe supports three services now!
+
+
+<!-- more -->
+
+
+## Comment support
+[@yausername](https://github.com/yausername) implemented one of the most requested features: support for reading comments!
+
+From now on, NewPipe can display comments underneath the video description. At the moment comment support is limited to reading top-level comments. But [@yausername](https://github.com/yausername) plans to extend it when he has some spare time left.
+
+Nevertheless, NewPipe can and will only support reading comments, because any other activity like writing or up-/down-voting comments require a YouTube account. For more on this topic, take a look at our [FAQ](/FAQ/bugs/).
```

Comment:
Either "activities [...] require" or "activity [...] requires".

```suggestion
Nevertheless, NewPipe can and will only support reading comments, because any other activity like writing or up-/down-voting comments requires a YouTube account. For more on this topic, take a look at our [FAQ](/FAQ/bugs/).
```